src/cmd/manage_nsfs.js

```diff
@@ -33,6 +33,7 @@ let config_fs;
 
 async function main(argv = minimist(process.argv.slice(2))) {
     try {
+        manage_nsfs_validations.validate_no_leading_whitespace(process.argv.slice(2));
```

Comment:
when doing it on the arguments we won't handle --from_file option

Reply:
We don't need to validate arguments we are getting from file as they should be in JSON format already